src/Window.cpp

```diff
@@ -3,6 +3,8 @@
 #include "CSS.h"
 #include "Wayland.h"
 
+#include <wayland-server-protocol.h>
```

Comment:
I'm not particularly fond of the inclusion of a wayland header into Window.cpp, as it shouldn't be concerned with the implementation details. I'd prefer if ```Wayland::Monitor``` would just contain an integer ```rotation``` and then ```GetWidth```/```GetHeight``` could just match against 90/270